src/components/menu/main-menu.tsx

```diff
@@ -110,86 +191,102 @@ const MenuItem = ({id, url, title, activeTrail, children, level}: MenuItemProps)
+      "border-transparent": level === 0 && !isCurrent && !inTrail,
+    },
+  );
 
   const subMenuStyles = clsx(
-    "list-unstyled w-full min-w-[300px] lg:bg-white lg:shadow-2xl px-0 lg:absolute",
+    "list-unstyled w-full min-w-[350px] lg:bg-fog-light lg:border lg:border-white lg:shadow-2xl lg:absolute lg:rounded-[25px] lg:px-12 lg:py-4 lg:mt-3",
     zIndexes[level],
     {
       "lg:top-full lg:right-0": level === 0,
       "lg:top-0": level !== 0,
       "lg:left-full": level !== 0 && positionRight,
       "lg:right-full": level !== 0 && !positionRight,
-      "block": submenuOpen,
-      "hidden": !submenuOpen,
+      block: submenuOpen,
+      hidden: !submenuOpen,
     }
-  )
+  );
 
   return (
     <li
       ref={menuItemRef}
-      className={clsx("m-0 py-2 lg:py-0 relative border-b first:border-t last:border-0 border-cool-grey lg:border-black-20 lg:relative lg:mr-5 last:lg:mr-0", {"lg:border-b-0 first:border-t-0": level === 0})}
+      className={clsx(
+        "m-0 lg:py-0 relative border-b last:border-0 border-spirited-light lg:relative",
+        { "lg:border-b-0 first:border-t-0 lg:mr-10 lg:last:mr-0": level === 0 }
+      )}
     >
-      <div className="flex items-center justify-between lg:justify-end">
+      <div className="group flex lg:justify-end">
         <Link
           id={linkId}
           href={url || "#"}
           className={linkStyles}
           aria-current={isCurrent ? "true" : undefined}
         >
           {title}
+          {level !== 0 && <ArrowRightIcon width={20} className="hidden group-focus:inline-block group-hover:inline-block ml-2" />}
         </Link>
 
-        {(children.length > 0 && level < menuLevelsToShow) &&
-          <>
-            {level === 0 && <div className="block ml-5 w-[1px] h-[25px] mb-[6px] bg-archway-light shrink-0"/>}
-            <button
-              ref={buttonRef}
-              className="shrink-0 relative right-10 lg:right-0 text-white lg:text-digital-red bg-digital-red lg:bg-transparent rounded-full lg:rounded-none group border-b border-transparent hocus:border-black hocus:bg-white"
-              onClick={toggleSubmenu}
-              aria-expanded={submenuOpen}
-              aria-labelledby={linkId}
-            >
-              <ChevronDownIcon
-                height={35}
-                className={clsx("transition group-hocus:scale-125 group-hocus:text-black ease-in-out duration-150", {"rotate-180": submenuOpen})}
-              />
-            </button>
-
-          </>
+        {children.length > 0 && level < menuLevelsToShow &&
+          <button
+            ref={buttonRef}
+            className={chevronBtnStyles}
+            onClick={toggleSubmenu}
+            aria-expanded={submenuOpen}
+            aria-labelledby={linkId}
+          >
+            <ChevronDownIcon
+              height={25}
+              className={clsx(
+                "transition group-hocus:scale-125 group-hocus:text-black ease-in-out duration-150",
```

Comment:
Need some visual change besides only transition for mobile. right now mobile has no underline or anything.